Sources/EduIDExpansion/Controllers/Onboarding/Coordinator/CoordinatorType.swift

```diff
@@ -11,6 +11,7 @@ protocol CoordinatorType: AnyObject {
     
     var navigationController: UINavigationController { get set }
     var children: [CoordinatorType] { get set }
-    func showNextScreen(currentType: ScreenType)
+    func showNextScreen(currentScreen: ScreenType)
+    func goBack()
```

Comment:
These seem specific to an OnboardingCoordinator, not to all coordinators? Or should this be called OnboardingCoordinatorType?

Reply:
I made a separate OnboardingCoordinatorType